### PR TITLE
Include buttercup specs in coveralls report

### PR DESCRIPTION
**Pull request by luisgerhorst**

### CODE REVIEW

**Comment by luisgerhorst** [comment]
Sorry for the spamming, that's it.

File: .travis.yml [hunk 1]
```diff
@@ -30,7 +30,10 @@ install:
 script:
   - make compile
   - make lint
+  - make clean # Required for working test coverage report.
   - make specs
+  # Workaround for 24.5, see https://github.com/cask/cask/issues/241
+  - find .cask -name "*.elc" | xargs rm
```
Comment (luisgerhorst) [inline .travis.yml:36]: As they say in the issue this seems to prevent a bug in ert-runner

File: test/specs/undercover-init.el [hunk 2]
```diff
@@ -1,7 +1,5 @@
 (when (require 'undercover nil t)
   (undercover "*.el"
-              (:exclude "*test*.el")
```
Comment (luisgerhorst) [inline test/specs/undercover-init.el:3]: I think undercover does not search subdirectories so this does not seem to be necessary.

**Comment by nloyola** [comment]
Thanks for cleaning this up. It is much better now.

**Pull request merged**